sources/packages/api/src/database/entities/student.model.ts

```diff
@@ -54,4 +54,16 @@ export class Student extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   user: User;
+
+  @Column({
+    name: "pd_date_sent",
+    nullable: true,
+  })
+  StudentPDSentAt?: Date;
```

Comment:
Please follow the same naming conventions from others properties starting the names with lower case.